tensorflow_serving/model_servers/main.cc

```diff
@@ -265,6 +272,42 @@ class PredictionServiceImpl final : public PredictionService::Service {
   bool use_saved_model_;
 };
 
+class ModelManagement final : public ModelManagementService::Service {
+  public:
+    explicit ModelManagement(ServerCore *core)
+            : core_(core) {}
+
+    grpc::Status Load(ServerContext *context,
+                      const LoadModelConfigurationRequest *request,
+                      LoadModelConfigurationResponse *response) override {
+      const tensorflow::serving::ModelConfigList &list = request->configuration().model_config_list();
```

Comment:
Please use a switch statement to handle the various "oneof" cases, and throw an error if it's any case other than ModelConfigList.

Reply:
Do you mean gRPC anyof field type? The request is using single type. Or I'm just missing something

Reply:
No. The ModelServerConfig proto has a one-of in it. The ModelConfigList is only one possible thing that might be in there (but the only one we need to handle at this time).

>>>
oneof config {
    ModelConfigList model_config_list = 1;
    google.protobuf.Any custom_model_config = 2;
  }
<<<